tests/behaviour/steps/concept/thing/attribute.rs

```diff
@@ -39,7 +39,8 @@ async fn attribute_put_instance_with_value(
     value: params::Value,
     may_error: params::MayError,
 ) {
-    may_error.check(&attribute_put_instance_with_value_impl(context, type_label, value));
+    may_error
+        .check_concept_write_without_read_errors(&attribute_put_instance_with_value_impl(context, type_label, value));
```

Comment:
Btw, couldn't come up with a better explicit name. I have to specify the error type as we need to write different logic for different error types, but it ends up being really verbose.